apps/nextjs-website/react-components/components/Header/Header.helpers.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, useTheme, type Theme } from '@mui/material';
+import { DialogBubbleProps } from '../../types/Header/Header.types';
+
+const useStyles = (mui: Theme) => ({
```

Comment:
```suggestion
const useStyles = (muiTheme: Theme) => ({
```
For consistency with other similar functions